__tests__/integration/integration.test.ts

```diff
@@ -14,24 +10,45 @@ import {
   StartTrigger,
   TenantedDeploymentMode
 } from '@octopusdeploy/message-contracts'
-import {RunConditionForAction} from '@octopusdeploy/message-contracts/dist/runConditionForAction'
-import {RunbookEnvironmentScope} from '@octopusdeploy/message-contracts/dist/runbookEnvironmentScope'
-import {setOutput} from '@actions/core'
-import {OctopusCliWrapper} from '../../src/octopus-cli-wrapper'
+import { RunConditionForAction } from '@octopusdeploy/message-contracts/dist/runConditionForAction'
+import { RunbookEnvironmentScope } from '@octopusdeploy/message-contracts/dist/runbookEnvironmentScope'
```

Comment:
We should only import from `@octopusdeploy/message-contracts`. We don't want to leak internals from `dist`.

Reply:
Done, thanks